src/components/Modal/index.jsx

```diff
@@ -198,7 +198,7 @@ export default class Modal extends React.Component {
     }
 
     return (
-      <Portal closeOnEsc isOpened onClose={this.cancel}>
```

Comment:
@masylum it closes, but the `cancel` function is not called. 👈 this sucks.
@andresgutgon I don't understand this solution. Could you explain?